website/docs/core-concepts/organisations.md

```diff
@@ -4,4 +4,11 @@ sidebar_position: 8
 
 # Organisations
 
-## Coming Soon!
+To access Dega, creating an organization is mandatory. If Super Organisation is created while installing Dega, then the user can access Dega or else needs to create an organization from Kavach for having an access to Dega.
+There is only one Super Organisation for a Dega instance but can create multiple Organisations as per user requirements. Applications can be added to an organization. 
```

Comment:
Change the rest of the page to be more descriptive and check on the grammar. For eg: _Applications can be added to an organization._ seems like a random statement if no context is provided on what an `application` is.